lib/hyperion/formats.rb

```diff
@@ -18,6 +18,7 @@ def write(obj, format)
       case Formats.get_from(format)
       when :json; write_json(obj)
       when :protobuf; obj
+      when :form_data; obj
```

Comment:
So the usage would become:

```ruby
body = Multipart.new(file: File.open('std1.mzdata.xml'), 'r'))
route = RestRoute.new(:post,
                      'http://10.10.5.106/convert',
                      ResponseDescriptor.new('converted_sample', 1, :json))
Hyperion.request(route, body)
```

No `PayloadDescriptor` needs to be specified because we can't translate it to a Typhoeus capability.